distribution/lib/Standard/Base/0.0.0-dev/src/Enso_Cloud/Enso_File_System_Impl.enso

```diff
@@ -0,0 +1,9 @@
+private
+
+import project.System.File.File_System_SPI
+import project.Enso_Cloud.Enso_File
+
+type Enso_File_System_Impl
+
+File_System_SPI.from (_:Enso_File_System_Impl) =
```

Comment:
To ensure this works, you should remove [EnsoPathFileSystemImpl.java](https://github.com/enso-org/enso/blob/498b5062a6f9fae70dba9e7004fd11e207fc809a/std-bits/base/src/main/java/org/enso/base/enso_cloud/EnsoPathFileSystemImpl.java)?